src/otx/algorithms/common/adapters/torch/amp/xpu_grad_scaler.py

```diff
@@ -0,0 +1,117 @@
+"""Custom GradScaler to scale loss."""
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from collections import abc, defaultdict
+from typing import List
+
+import intel_extension_for_pytorch as ipex  # noqa: F401
```

Comment:
It's unused package. How aboug removing it?

Reply:
Done.